app/helpers/links_helper.rb

```diff
@@ -4,6 +4,7 @@ module LinksHelper
     'Crimethinc. on Facebook'          => 'https://www.facebook.com/To-Change-Everything-103734818122357',
     'CrimethInc. on Github'            => 'https://github.com/crimethinc',
     'CrimethInc. on Tumblr'            => 'https://crimethinc.tumblr.com',
+    'Crimethinc. on bandcamp'          => 'https://crimethinc.bandcamp.com',
```

Comment:
```suggestion
    'CrimethInc. on Bandcamp'          => 'https://crimethinc.bandcamp.com',
```

Reply:
I let `bandcamp` in lowercase because on their site, their logo is all in lower case.

Reply:
The way I always check is to look at the plaintext version in the `<title>` tag.